d2/src/main/java/com/linkedin/d2/xds/XdsChannelFactory.java

```diff
@@ -79,6 +103,15 @@ public ManagedChannel createChannel()
     {
       _log.info("Applying custom load balancing policy for xDS channel: {}", _defaultLoadBalancingPolicy);
       builder = builder.defaultLoadBalancingPolicy(_defaultLoadBalancingPolicy);
```

Comment:
This would be `null` by default? Does that work?

Reply:
```java
    if (_defaultLoadBalancingPolicy != null)
    {
      _log.info("Applying custom load balancing policy for xDS channel: {}", _defaultLoadBalancingPolicy);
      builder = builder.defaultLoadBalancingPolicy(_defaultLoadBalancingPolicy);
```
This code block is only entered if it _isn't_ null.

By default it uses `pick_first` though